core/unit_test/TestAtomics.hpp

```diff
@@ -97,7 +97,7 @@ struct SuperScalar {
   }
 
   KOKKOS_INLINE_FUNCTION
-  SuperScalar operator+(const SuperScalar& src) {
+  SuperScalar operator+(const SuperScalar& src) const {
```

Comment:
What's the deal with that change?

Reply:
This was missing, and the desul atomic implementation noticed that complaining about a const overload missing for the function.